docs/api/en/helpers/BoxHelper.html

```diff
@@ -28,10 +28,25 @@ <h2>Example</h2>
 
 
 		<code>
-		var sphere = new THREE.SphereGeometry();
-		var object = new THREE.Mesh( sphere, new THREE.MeshBasicMaterial( 0xff0000 ) );
-		var box = new THREE.BoxHelper( object, 0xffff00 );
-		scene.add( box );
+		// Creating the object whose bounding box we want to compute
+		var sphereGeom = new THREE.SphereGeometry();
+		var sphereObject = new THREE.Mesh( sphereGeom, new THREE.MeshBasicMaterial( 0xff0000 ) );
+
+		// Creating the helper and displaying it in the scene
+		var helper = new THREE.BoxHelper( sphereObject, 0xffff00 );
+		scene.add( helper );
+
+		// Creating the actual bounding box (on which we can test intersection with other Box3 objects)
+		var box3 = new THREE.Box3();
+		// Shape and position the box after the object
+		box3.setFromObject( sphere );
```

Comment:
I think it should be `sphereObject` and not `sphere`.

Reply:
Yeah, my bad. It was a typo.